setup.py

```diff
@@ -11,11 +11,13 @@
       packages=['aw_core', 'aw_datastore'],
       install_requires=[
           'jsonschema',
+          'peewee>=2.8.5',
           'strict-rfc3339',
           'appdirs==1.4.0',
           'nose-parameterized',
           'iso8601',
           'tinydb>=3.2.1',
           'tinydb-serialization>=1.0.3',
           'python-json-logger>=0.1.5',
+          'takethetime>=0.3.0',
```

Comment:
Yes, only for benchmarking. It is a very small pure-python package I wrote. Nothing like "install-develop" afaik.